legend-engine-test-runner-mapping/src/test/java/org/finos/legend/engine/test/runner/mapping/TestMappingTestRunner.java

```diff
@@ -293,4 +106,93 @@ public void jsonDeserializationTest() throws IOException
         PureModelContextData pureModelContextData = ObjectMapperFactory.getNewStandardObjectMapperWithPureProtocolExtensionSupports().readValue(url, PureModelContextData.class);
         new PureModel(pureModelContextData, null, Thread.currentThread().getContextClassLoader(), DeploymentMode.PROD);
     }
+
+    //testable tests
+    @Test
+    public void testMappingTestSuiteForM2MUsecase()
```

Comment:
Can we move testSuite tests to separate file under package
org.finos.legend.engine.testable.mapping.extension

Reply:
Resolved